benchmarks/benchmark_baselines.py

```diff
@@ -0,0 +1,33 @@
+"""Benchmarking for baselines."""
+import random
+
+from benchmarks import benchmark, iterate_experiments
+from benchmarks.experiments.baselines import continuous_mlp_baseline_tf
+from benchmarks.experiments.baselines import gaussian_cnn_baseline_tf
+from benchmarks.experiments.baselines import gaussian_mlp_baseline_tf
+
+
+@benchmark
+def ppo_continuous_mlp_baseline_tf_benchmarks():
+    """Run experiments for Continuous MLP Baseline TF benchmarking."""
+    seeds = random.sample(range(100), 3)
+    iterate_experiments(continuous_mlp_baseline_tf, seeds=seeds)
+
+
+@benchmark
+def ppo_gaussian_cnn_baseline_tf_benchmarks():
+    """Run experiments for Gaussian CNN Baseline TF benchmarking."""
+    env_ids = ['CubeCrash-v0', 'MemorizeDigits-v0']
```

Comment:
Since we already have the [environment list](https://github.com/rlworkgroup/garage/tree/master/tests/benchmarks) for benchmarking for different kind of algorithms, should we make it a global dict? (so we won't need to worry about typo for `env_ids` )

Reply:
Okey, let's do it in another PR. I need some clarification for this.

Reply:
Updated.